Codecademy/gamut
# feat: add listStyle props to Box and Column

### Overview


add listStyle, listStyleType, listStyleImage, and listStylePosition props to `Box` and `Column`



### PR Checklist

- [x] Related to JIRA ticket: [REACH-3045](https://codecademy.atlassian.net/browse/REACH-3045)

#### Testing Instructions



- add `listStyle`, `listStyleType`, `listStyleImage`, and `listStylePosition` props to `Column`, `Box`, `FlexBox`, and `GridBox` components
- ensure code passes type checker
- ensure prop values are rendered as styles for the component

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/3657)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-3657)   |




[REACH-3045]: https://codecademy.atlassian.net/browse/REACH-3045?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
